docs/content/docs/custom-resource/autoscaler.md

```diff
@@ -308,6 +308,13 @@ org.apache.flink.autoscaler.standalone.StandaloneAutoscalerEntrypoint \
 --autoscaler.standalone.jdbc.username root
 ```
 
+### Load StandaloneAutoscaler configuration through configuration file
+
+Can specify the running configuration of StandaloneAutoscaler in the configuration file. The config.yaml and
+flink-conf.yaml files in the environment variable `FLINK_CONF_DIR` directory will be loaded by default. You can also pass
+`export CONF_OVERRIDE_DIR=xxx` to specify the loaded configuration file directory.
+NOTE: main configuration will overwrite the configuration in the configuration file
```

Comment:
How about update the `main configuration` to `main parameters`? 

I saw most of doc saw the Java Main parameters. WDYT?

Reply:
> How about update the `main configuration` to `main parameters`?
> 
> I saw most of doc saw the Java Main parameters. WDYT?

+1.